drivers/i3c/master/mipi-i3c-hci/core.c

```diff
@@ -747,7 +747,8 @@ static int i3c_hci_attach_i3c_dev(struct i3c_dev_desc *dev)
 		return -ENOMEM;
 	if (hci->cmd == &mipi_i3c_hci_cmd_v1) {
 #ifdef CONFIG_ARCH_ASPEED
-		ret = mipi_i3c_hci_dat_v1.alloc_entry(hci, dev->info.dyn_addr);
+		ret = mipi_i3c_hci_dat_v1.alloc_entry(hci,
+						      dev->info.dyn_addr ?: dev->info.static_addr);
```

Comment:
Please use full form of Ternary Condition.
(condition)? True : false;

Reply:
(dev->info.dyn_addr != NULL) ? dev->info.dyn_addr : dev->info.static_addr;

Reply:
> (dev->info.dyn_addr != NULL) ? dev->info.dyn_addr : dev->info.static_addr;

fixed it. please check the new commits